components/button/Button.module.css

```diff
@@ -1,8 +1,10 @@
 .btn {
   border-radius: 10px;
+  font-family: var(--BodyFont);
+  font-size: 30px;
```

Comment:
These should be responsive

Reply:
would be fixed in next pullrequest. thank you